en/docs/migrate-to-choreo/how-to-migrate-to-choreo.md

```diff
@@ -0,0 +1,27 @@
+# How to Migrate to Choreo?
+
+## Migration process
+
+-  It is not possible to automate the migration process due to several reasons. WSO2 API Cloud and Choreo are two distinct platforms with different architectures, making an automated process too complicated. Additionally, some API Cloud customers have customizations (such as custom domains and user stores) that require manual migration.
```

Comment:
process "is" too complicated ?

Reply:
Rephrased the sentence to improve clarity.